tracer/test/snapshots/NetActivitySdkTests.verified.txt

```diff
@@ -13,11 +13,16 @@
       otel.status_code: STATUS_CODE_UNSET,
       otel.trace_id: Guid_1,
       runtime-id: Guid_2,
+      some_tag: value,
       span.kind: server,
-      version: 1.0.0
+      version: 1.0.0,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
+      _dd.git.repository_url: https://github.com/DataDog/dd-trace-dotnet,
+      _dd.span_links: [{"span_id":"span_link_id","trace_id":"trace_link_id","attributes":{"some_int":5,"some_string":"five","some_bool":false,"some_int[]":[5,55,555]},"dropped_attributes_count":"2","flags":"0"},{"span_id":"span_link_id","trace_id":"trace_link_id","flags":"1"}]
```

Comment:
Just wanted to double check whether arrays were accurate - example is `some_int[]` shows up as `[5,55,555]` do we want it like that or did we need to flatten these?

Reply:
This will get remove when I updated the snapshots and look like the new value, will add a link to the relevant commit in a few.